### PR TITLE
javascript-javascript2-week1

### PR DESCRIPTION
**Pull request by Hailegebriel22**

### CODE REVIEW

File: javascript/javascript2/week1/week1Js2.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        }
+        else {
+            shortestWord = danishWords[i++]
+        }
+    }
+    return shortestWord;
+} 
+
+
+console.log(shorterword(danishWords))  // returns 'ø'
```
Comment (shravanipagilla) [inline javascript/javascript2/week1/week1Js2.js:18]: Code looks fine except a little changes need to be done.
1. seems this function returning different word in different runs.  seems there might need to make little changes to for loop. 
reference: function shorterword(danishWords) {
    let shortestWord = danishWords[0];                   //little change here
    for (let i = 0; i < danishWords.length; i++) {
        if (danishWords[i].length < shortestWord.length) {.  // small change here
            shortestWord = danishWords[i]; 
        }
    }
  
    return shortestWord;
} 

//no need of else statement.
You can also do it many other ways , this is just for your reference.

Comment (shravanipagilla) [inline javascript/javascript2/week1/week1Js2.js:18]: As the basic requirement , this output need to be printed in HTML using DOM manipulation.

File: javascript/javascript2/week1/week1Js2.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        else if (danishString[i] === 'ø' || danishString[i] === 'Ø') {
+            count3 += 1;
+        }
+        total = count1 + count2 + count3;
+
+    }
+    return `{total: ${total}, æ:${count2}, ø:${count3}, å:${count1} } `
+
+}
+console.log(DanishLettersCounter(danishString2))
```
Comment (shravanipagilla) [inline javascript/javascript2/week1/week1Js2.js:51]: As the basic requirement , this output need to be printed in HTML using DOM manipulation.

**Review by shravanipagilla — COMMENTED** [review]
Over all , I can see DOM manipulation overriding by each other. 
For example Product info is displaying using UL and immediately overridden by random name function.

**Comment by Hailegebriel22** [comment]
> Over all , I can see DOM manipulation overriding by each other. For example Product info is displaying using UL and immediately overridden by random name function.

@shravanipagilla thanks for the comment but I don't understand this comment. The product list generates a random list of products every time it runs.

**Comment by shravanipagilla** [comment]
Hi again,
It is just simple thing, that you are using 'ul' to Print products looks good. 
But this 'ul' is a part inside <div id='name'> ,  you are also using this div to print 'spirit Animal Generator'.

Both are working fine but i can't see both at the same time, this is due to you are using same place on HTML page to print both.  
Suggestion: use different <div id='something'> for spirit animal function.  if you did't understand this one i will explain in sunday class . i hope you can understand .:)

**Pull request merged**